Angular/src/app/app.component.html

```diff
@@ -0,0 +1,16 @@
+<div class="container">
+  <app-custom-scheduler
```

Comment:
Let's remove this scheduler widget wrapper and use it directly.

```html
<div class="container">
  <dx-scheduler
    class="scheduler"
    currentView="month"
    [dataSource]="dataSource"
    [views]="['month']"
    [currentDate]="currentDate">
  </dx-scheduler>
  <!-- Other code -->
</div>
```

And in app.component.css:
```css
.container {
  display: flex;
}

.scheduler {
  display: block;
  width: 80%;
}
```